modules/creating_a_module.md

```diff
@@ -72,88 +80,51 @@ jhipsterFunc.registerModule(npmPackageName, hookFor, hookType[, callbackSubGener
-- `addMainSCSSStyle`: add a new style in the `main.scss` file
-- `addAngularJsModule`: add a new module in the `app.js` file
-- `addAngularJsInterceptor`: register an AngularJS interceptor in the `app.js` file
-- `addElementToMenu`: add an entry in the navigation menu
-- `addElementToAdminMenu`: add an entry in the admin navigation sub-menu
-- `addEntityToMenu`: add an entity in the entity navigation sub-menu
-- `addElementTranslationKey`: add a new translation key in the `global.json` file
-- `addAdminElementTranslationKey`: add a new translation key for an admin sub-menu in the `global.json` file
-- `addEntityTranslationKey`: add a new translation key for an entity in the `global.json` file
-- `addGlobalTranslationKey`: add a new translation key in the `global.json` file
-- `addTranslationKeyToAllLanguages`: add a new translation key for all installed languages using methods `addElementTranslationKey`, `addEntityTranslationKey`, `addAdminElementTranslationKey`
-- `isSupportedLanguage`: check if a language is supported by JHipster
-- `getAllSupportedLanguages`: get the list of languages supported by Jhipster
-- `getAllSupportedLanguageOptions`: get all the language options supported by JHipster
-- `getAllInstalledLanguages`: get the list of languages installed by current application
-- `addChangelogToLiquibase`: add a new changelog in the Liquibase `master.xml` file for a given needle
-- `addConstraintsChangelogToLiquibase`: add a new constraints changelog to the Liquibase `master.xml` file
-- `addLiquibaseChangelogToMaster`: add a new changelog to the Liquibase `master.xml` file
-- `addColumnToLiquibaseEntityChangeset`: add new columns to the Liquibase changelog for an entity
-- `dateFormatForLiquibase`: creates a new timestamp to be used by a Liquibase changelog
-- `copyI18nFilesByName`: copy i18n files
-- `copyTemplate`: copy a template from source to a destination after stripping any translation content when translation is disabled
-- `copyHtml`: short hand method for `copyTemplate` which is defaulted to action `stripHtml`
-- `copyJs`: short hand method for `copyTemplate` which is defaulted to action `stripJs`
-- `rewriteFile`: add the given content above a specific custom needle in a file
-- `replaceContent`: replace the given content for a specific pattern/regex in a file
-- `registerModule`: register a JHipster module to act as a hook from app or entity generator
-- `getModuleHooks`: get the array of all registered module hooks for the application
-- `updateEntityConfig`: update the json configuration file for an entity with given key and value
-- `getExistingEntities`: get sorted list of entities according to changelog date in the current application
-- `isJhipsterVersionLessThan`: check if the JHipster version used to generate the project is less than a particular version
+## Available variables and functions
+
+### Variables from configuration:
 
+You have to use this function:
+
+```
+    _getConfig() {
```

Comment:
I think we can provide this function in generator base, so users can just use it